package.json

```diff
@@ -63,20 +63,18 @@
     "mocha": "^9.2.2",
     "mocha-sinon": "^2.1.2",
     "nyc": "^15.1.0",
-    "prettier": "^2.7.1",
-    "rimraf": "^3.0.2",
-    "semver": "^7.3.8",
+    "prettier": "^2.8.8",
+    "semver": "^7.5.0",
     "sinon": "^13.0.1",
     "sinon-chai": "^3.7.0",
     "source-map-support": "^0.5.21",
     "standard-version": "^9.5.0",
     "ts-node": "^10.9.1",
-    "tsd": "^0.24.1",
+    "tsd": "^0.27.0",
```

Comment:
Did we also intend to bump the tsd version in 4.x as well? I'm pretty sure we did, but I just wanted to make sure since it wasn't in this ticket's description.